main/persistent_storage/persistent_storage.h

```diff
@@ -13,6 +13,8 @@
 
 #define WIFI_SSID_KEY "WIFI_SSID"
 #define WIFI_PASSWORD_KEY "WIFI_PASSWORD"
+#define SENSOR_ID_KEY "SID"
```

Comment:
Ander format dan bovenstaande, zou S_ID moeten zijn

Reply:
SID is an abbreviation on it's own, WIFI_SSID are two seperate words, this is fine